test/e2e/e2e_test.go

```diff
@@ -358,21 +340,20 @@ var _ = Describe("E2E Tests", func() {
 			Expect(len(nodesWithImage)).Should(Equal(9))
 		})
 		It("Should load multiple images to cluster 1 and 3 only", func() {
-			flags := &loadimagecmd.LoadImagesFlagpole{
-				Debug:  true,
-				Images: []string{"nginx:stable-alpine", "alpine:latest"},
-				Clusters: []string{
-					defaults.ClusterNameBase + strconv.Itoa(1),
-					defaults.ClusterNameBase + strconv.Itoa(3),
-				},
-			}
 			log.SetLevel(log.DebugLevel)
+
+		    images := []string{"nginx:stable-alpine", "alpine:latest"}
```

Comment:
I think this line should be indented one more level.

Reply:
Done. It looked correct in GoLand - I guess one of the indentations must have had spaces and not a tab.